Dockerfile

```diff
@@ -18,6 +18,8 @@ RUN adduser -D as-user
 USER as-user
 COPY --chown=as-user --from=builder /opt/auth-service .
 # cleanup
-RUN npm prune --production
+
+# disable until problems with TS->JS transpilation solved
+# RUN npm prune --production
```

Comment:
or move ts-node to prod dependencies...

Reply:
@eoln Turns out that this line also breaks integration tests because axios is also a devDependency. 
Moving axios to prod dependencies doesn't make sense since it is only used for testing.